PyDSS/pydss_results.py

```diff
@@ -48,7 +51,11 @@ def __init__(
         """
         options = ElementOptions()
         if project_path is not None:
-            self._project = PyDssProject.load_project(project_path)
+            # TODO: handle old version?
```

Comment:
This will likely not work if a user tries to load the results of an existing project. We could add backward compatibility or make the user install the previous version in order to load it. What do you think?